cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1258,8 +1299,203 @@ def _add_head_node(self):
                 properties={"ConfigVersion": self._cluster_config.config_version, "DynamoDBTable": ""},  # TODO
             )
 
-    def _head_node_efa_enabled(self):
-        return self._cluster_config.head_node.efa.enabled if self._cluster_config.head_node.efa else "NONE"
+    def _add_compute_fleet(self):
```

Comment:
Any thoughts on sharing some code with `_add_head_node()`?

Reply:
Yes, but it's not so easy because compute-fleet merges information from both queue and compute.
For sure we have a lot of things that can be refactored here, but I'm trying to reach an end-to-end flow, then we'll refactor and fix all the small things. Thanks for pointing this out.